File: week-2/twoSum.py
def twoSum(nums, target):
    # your code here
    for i, element in enumerate(nums):
        lack = target - element
        if lack in nums:
            return [i, nums.index(lack)]


result = twoSum([2, 11, 7, 15], 9)
print(result)  # show [0, 2] because nums[0]+nums[2] is 9

assert result == [0, 2]

File: week-2/twoSum.js
function twoSum(nums, target) {
    // your code here
    let i;
    for (i = 0; i < nums.length; i++) {
        let idx = nums.indexOf(target - nums[i]);
        if (idx != -1) {
            return [i, idx]
        }
    }
}
let result = twoSum([2, 11, 7, 15], 9);
console.log(result); // show [0, 2] because nums[0]+nums[2] is 9

console.assert(JSON.stringify(twoSum([2, 11, 7, 15], 9)) === JSON.stringify([0, 2]), 'Wrong return.');